src/components/contact_id.js

```diff
@@ -0,0 +1,49 @@
+import { BrowserRouter, Switch, Route, Link, useParams } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const ContactId = (props) => {
+  const passedContacts = props.passFromMain;
```

Comment:
You can just call this `contacts`, it's evident that they're coming from elsewhere by virtue of them being props